Habicus/Habicus-Core-Web
# HABI-48 Adding better persistence logic for table structure

- Redesigned table structure and removed old code

![](https://camo.githubusercontent.com/cf373500dfe17773807191f7fb730ea7ea4b959d/68747470733a2f2f692e696d6775722e636f6d2f5878585a6b6d4f2e706e67)


#### Ticket Reference ([View Current Issues Here](https://github.com/Habicus/Habicus-Core/issues))
My Issue Number Connects #48 

##### Backlog & Current Sprint ([View Backlog Here](https://waffle.io/Habicus/Habicus-Core-Web))


##### Core Reviewers:
@Habicus/core-team

##### Merge Owner:
@Habicus/core-team
@austinsorrells
@schachte

## Status
**WIP**

## Description
Replaces old data loader strategy with table structure that makes sense and auto creates table during initialization.

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
Deprecating Data Loader | #46 


## Completed The Following (If Relevant)

## Steps to Test or Reproduce
1. Do a full build on dev profile
2. Check the embedded database to ensure that the tables were created successfully
